package.json

```diff
@@ -4,10 +4,13 @@
   "description": "",
   "main": "server/index.js",
   "scripts": {
+    "postinstall": "npm run build",
```

Comment:
Dont include both `build` and `postinstall` for doing the same job! Heroku will run twice.